app/models/incident/IncidentCode.scala

```diff
@@ -16,48 +16,17 @@
 
 package models.incident
 
-import models.{EnumerableType, Radioable, WithName}
+import models.{DynamicEnumerableType, Radioable}
+import play.api.libs.json.{Format, Json}
 
-sealed trait IncidentCode extends Radioable[IncidentCode] {
-  override val messageKeyPrefix: String = IncidentCode.messageKeyPrefix
-  val code: String
-}
-
-object IncidentCode extends EnumerableType[IncidentCode] {
-
-  case object DeviatedFromItinerary extends WithName("deviatedFromItinerary") with IncidentCode {
-    override val code: String = "1"
-  }
-
-  case object SealsBrokenOrTampered extends WithName("sealsBrokenOrTampered") with IncidentCode {
-    override val code: String = "2"
-  }
+case class IncidentCode(code: String, description: String) extends Radioable[IncidentCode] {
```

Comment:
I was wondering if we should modify this to `case class IncidentCode(code: String, description: String)` to case class IncidentCode(code: Incident, description: String)

Where Incident is the same as before:
sealed trait Incident
case object DeviatedFromItinerary extends Incident
...

The way we have it now means the json parsing is easier but it feels strange matching on magic strings.
Maybe we can look at this in a later refactor, I know I have followed the same pattern in my pr.